USAGE_android.md

```diff
@@ -234,176 +204,38 @@ It may also be set as a post attach command in the project configuration:
-
-#### Copy the Capture Layer Onto the Device
-
-The data folder returned by the instructions in
-[Locate the Application Data Directory](#locate-the-application-data-directory)
-above can be used to store the layer, but it must be placed in the appropriate
-library path.
-However, you can't copy it directly, so you need to first push the file into
-either the `/sdcard/Download` or `/storage/emulated/0/Download` directory with an adb
-command:
-
-```bash
-adb push \
-    ./android/layer/build/intermediates/cmake/${Build Type}/obj/${Build Target}/libVkLayer_gfxreconstruct.so \
-    /sdcard/Download
-```
-
-For example, for a debug build of the ARM64 version of the library:
-
-```bash
-adb push \
-    ./android/layer/build/intermediates/cmake/debug/obj/arm64-v8a/libVkLayer_gfxreconstruct.so \
-    /sdcard/Download
-```
-
-Then move it to the application folder using the adb shell with Admin authority:
-
-```bash
-mv /sdcard/Download/libVkLayer_gfxreconstruct.so \
-       ${Application Data Path}/lib/arm64/libVkLayer_gfxreconstruct.so
-```
-
-Make the layer copied over above writeable on the system using the ADB
-Admin shell:
-
-```bash
-chmod 777 ${Application Data Path}/lib/arm64/libVkLayer_gfxreconstruct.so
-```
-
-#### Give Application Write Permissions
-You may also have to give the application permission to write to external
-storage using:
+#### Install the Replay APK
 
-```bash
-adb shell pm grant ${Application Full Name} android.permission.WRITE_EXTERNAL_STORAGE
-```
-
-### Enabling the Layer with ADB
+The GFXReconstruct capture layer is included in the replay tool APK. Follow the [Install APK Command](#install-apk-command) instructions to make it available on the target device.
 
-#### Enable the Layer a Specific Application
+#### Enable the Layer for a Specific Application
 
-To enable the GFXReconstruct capture layer just for a specific application
-requires that 4 steps be taken:
+There are 5 steps to enable the GFXReconstruct capture layer for a capture target application:
 
- 1. Make sure the application is debuggable
+ 1. Make sure the target application is debuggable
  2. Enable GPU Debug layers
  3. Indicate what app you are enabling GPU Debug on
- 4. Identify the layers used to debug (in this case the layer is
-    `VK_LAYER_LUNARG_gfxreconstruct`)
+ 4. Identify the layers used to debug (in this case the layer is `VK_LAYER_LUNARG_gfxreconstruct`)
+ 5. Indicate what package to load debug layers from (in this case the package is `com.lunarg.gfxreconstruct.replay`)
 
-Together, the last 3 commands look like the following.
+Together, the last 4 commands look like the following:
 
 ```bash
 adb shell settings put global enable_gpu_debug_layers 1
-adb shell settings put global gpu_debug_app ${Application Full Name}
+adb shell settings put global gpu_debug_app ${Package Name}
 adb shell settings put global gpu_debug_layers VK_LAYER_LUNARG_gfxreconstruct
+adb shell settings put global gpu_debug_layers_app com.lunarg.gfxreconstruct.replay
```

Comment:
What versions of Android is this available in?

Reply:
As far as I can tell, this has been available since Android 10